src/figvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/applicant/ApplicantControllerValidateApplicantServiceTest.java

```diff
@@ -0,0 +1,192 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.applicant;
+
+import ca.bc.gov.open.ords.figcr.client.api.model.ValidateApplicantServiceOrdsResponse;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.applicant.ApplicantController;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.applicant.ApplicantServiceImpl;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.applicant.types.ValidateApplicantServiceResponse;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.exception.FigaroValidationServiceException;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ApplicantControllerValidateApplicantServiceTest {
+
+    // test class for ValidateApplicantService methods.
+
+    @Mock
+    private ApplicantServiceImpl applicantServiceMock;
+
+    private ValidateApplicantServiceResponse validateApplicantServiceResponse;
+
+    private ApplicantController sut;
+
+    /*
+     * Valid input arguments
```

Comment:
Do we need to describe why these are valid/invalid? Just wondering, might be useful in the future.

Reply:
That's a good point @akroon3r, future devs may look at these tests and have no idea of what we were doing. The fact that we've translated what is valid and invalid from webMethods may make decoding our work doubly hard when webMethods is decommissioned.

Reply:
so usually you put that in the name of the methods like withValidParamShouldSuccess, but we can discuss the level of doc we want in our unit tests